src/Evolution/Systems/Ccz4/DerivLapse.cpp

```diff
@@ -48,6 +49,37 @@ tnsr::ij<DataType, Dim, Frame> grad_grad_lapse(
                   christoffel_second_kind, field_a, d_field_a);
   return grad_grad_lapse_;
 }
+
+template <size_t Dim, typename Frame, typename DataType>
+void divergence_lapse(
+    const gsl::not_null<Scalar<DataType>*> div_lapse,
+    const Scalar<DataType>& conformal_factor,
+    const tnsr::II<DataType, Dim, Frame>& inverse_conformal_metric,
+    const tnsr::ij<DataType, Dim, Frame>& grad_grad_lapse) noexcept {
+  destructive_resize_components(div_lapse, get_size(get(conformal_factor)));
+  for (auto& component : *div_lapse) {
```

Comment:
isn't this for loop:
```cpp
get(div_lapse) = 0.0;
```
?

Reply:
Yes, I recycled this for loop from another function and didn't think about how it's not needed here